django_ninja_crudl/controller.py

```
@@ -79,21 +78,35 @@
-
         dynamic_class = api_controller(tags=config.tags)(
             type(
                 name,
                 final_bases,
                 dct,
             )
         )
-
         return dynamic_class
 
+    # TODO(phuongfi91): use this where it needed
```

Ruff (0.8.2)

Missing issue link on the line following this TODO

(TD003)